.zenodo.json

```
@@ -21,6 +21,11 @@
             "name": "Eiden, Kiran",
             "orcid": "0000-0001-6191-4285"
         },
+	{
+	    "affiliation": "Department of Physics and Astronomy, Stony Brook University",
+	    "name": "Sam Glosser",
+	    "orcid": "0009-0000-6648-3381"
+	},
```

tabs to spaces and fix name order
```suggestion
        {
            "affiliation": "Department of Physics and Astronomy, Stony Brook University",
            "name": "Glosser, Sam",
            "orcid": "0009-0000-6648-3381"
        },
```